11/jdk_11_0_24.json

```diff
@@ -7,10 +7,10 @@
             "openjdk_version_data": {
                 "major": 11,
                 "minor": 0,
-                "security": 23,
+                "security": 24,
                 "patch": 0,
                 "pre": null,
-                "build": 9,
+                "build": 8,
```

Comment:
Security increments but build decrements?

Reply:
This is a correction of the entry for the July 2024 PSU. Our published version for July is: `11.0.24+8`, which means that the build number should be `8`